librz/search/bytes_search.c

```diff
@@ -215,6 +216,7 @@ static bool bytes_find(RzSearchFindOpt *fopts, void *user, ut64 address, const R
 	const ut8 *raw_buf = rz_buf_get_whole_hot_paths((RzBuffer *)buffer, &size);
 	void **it = NULL;
 	RzPVector /*<BytesPattern *>*/ *patterns = (RzPVector *)user;
+	*num_hits = 0;
```

Comment:
why? `RzThreadQueue *hits` contains the hits, just invoke `rz_th_queue_size()` for the current size of the queue, i.e. the number of hits.

Reply:
`RzThreadQueue *hits` contains all of the hits that have been found so far in the search, not the number of hits in the current search interval